modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -0,0 +1,424 @@
+{rhel}{nbsp}8 clients are supported as Salt clients.
+====
+
+
+
+== Import Entitlements and Certificates
+
+{redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate, and an entitlement key.
+
+Entitlement certificates are embedded with expiration dates, which match the length of the support subscription.
+To avoid disruption, you will need to repeat this process at the end of every support subscription period.
+
+{redhat} supply a subscription manager tool to manage subscription assignments.
+It runs locally to track installed products and subscriptions.
+Clients must be registered with the subscription manager to obtain certificates.
+
+{redhat} clients use a URL to replicate repositories.
+The URL will change depending on where the {redhat} client is registered.
+
+{redhat} clients can be registered in three different ways:
+
+* {redhat} content delivery network (CDN) at redhat.com
+* {redhat} Satellite Server
+* {redhat} update infrastructure (RHUI) in the cloud
+
+This guide covers clients registered to {redhat} update infrastructure (RHUI).
+You must have at least one system registered to RHUI, with an authorized subscription for repository content.
+
+For information about using {redhat} content delivery network (CDN) instead, see xref:client-configuration:clients-rh-cdn.adoc[].
+
+
+[IMPORTANT]
+====
+Satellite certificates for client systems require a Satellite server and subscription.
+Clients using Satellite certificates are not supported with {productname} Server.
+====
+
+[WARNING]
+====
+Entitlement certificates are embedded with expiration dates, which match the length of the support subscription.
+To avoid disruption, you will need to repeat this process at the end of every support subscription period.
+====
+
+{redhat} supplies the subscription-manager tool to manage subscription assignments.
+It runs locally on the client system to track installed products and subscriptions.
+Register to redhat.com with subscription-manager, then follow this procedure to obtain certificates.
+
+
+
+.Procedure: Registering Clients to Subscription Manager
+
+. On the client system, at the command prompt, register with the subscription manager tool:
++
+----
+subscription-manager register
+----
++
+Enter your {redhat} Portal username and password when prompted.
+. Copy your entitlement certificate and key from the client system, to a location that the {productname} Server can access:
++
+----
+cp /etc/pki/entitlement/ /<example>/entitlement/
+----
++
+[NOTE]
+====
+Your entitlement certificate has a file extension of [path]``.crt``.
+For example, [path]``/etc/pki/rhui/product/content-rhel8.crt``.
+The key has a file extension of [path]``.key``.
+For example, [path]``/etc/pki/rhui/content-rhel8.key``.
+====
++
+. Copy the {redhat} CA Certificate file from the client system, to the same web location as the entitlement certificate and key:
++
+----
+cp /etc/rhsm/ca/redhat-uep.pem /example/entitlement
```

Comment:
er no, i think the last path is incorrect.

Here is my history of commands:
```
   12  scp /etc/pki/rhui/product/content-rhel7.crt 172.17.17.158:.
   13  scp /etc/pki/rhui/content-rhel7.key 172.17.17.158:.
   14  scp /etc/pki/rhui/cdn.redhat.com-chain.crt 172.17.17.158:.
```

File `/etc/rhsm/ca/redhat-uep.pem` exists, but I'm not sure it's of any use in the RHUI context.